packages/core/ReExports/modules.tsx

```diff
@@ -230,6 +232,8 @@ const libs = {
     useGridApiContext,
     useGridApiRef,
     useGridRootProps,
+    GridToolbar,
```

Comment:
could possibly make these into lazy imports. since it is a named import, you can synthesize a default export with an extra file, similar things done in e.g. https://github.com/GMOD/jbrowse-components/blob/main/packages/core/ReExports/Attributes.tsx